src/NIST-800-53/nist-800-53.ts

```diff
@@ -261,6 +290,169 @@ export class NIST80053Checks extends NagPack {
         this.createMessage(ruleId, info, explanation)
       );
     }
+    if (
+      !this.ignoreRule(
+        ignores,
+        'NIST.800.53-EC2CheckDefaultSecurityGroupClosed'
+      ) &&
+      !nist80053EC2CheckDefaultSecurityGroupClosed(node)
+    ) {
+      const ruleId = 'NIST.800.53-EC2CheckDefaultSecurityGroupClosed';
```

Comment:
This rule needs to be re-implemented. When creating a VPC in CloudFormation a default security group is created with the [following configuration](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/default-custom-security-groups.html#default-security-group) and is not present as a resource in the Cfn/CDK stack. Therefore we know that all VPCs do not have their default security group closed.

This rule should be reformatted to a warning telling users that they need to close the default security group after stack creation  any time a VPC is created.

Reply:
Ah, thanks for catching that.  I've re-implemented the rule as a warning, and have updated its tests and documentation in RULES.md.  Let me know if any further changes are required, thanks!